test/axios.test.js

```diff
@@ -709,7 +712,10 @@ describe("Axios AuthHttpRequest class tests", function() {
                 assertEqual(userId, loginResponse.data);
 
                 let data = await supertokens.getAccessTokenPayloadSecurely();
-                assertEqual(Object.keys(data).length, 0);
+
+                if (!jwtEnabled) {
+                    assertEqual(Object.keys(data).length, 0);
+                }
```

Comment:
Else it should be length 2?

Reply:
Done, it checks for 5 keys (`sub`, `iss`, `jwt`, `_jwtPName` and `customClaim`)